.codecov.yml

```diff
@@ -1 +0,0 @@
-comment: false
```

Comment:
I think there is no reason to delete this, I imagine even on github CI we should still do codecov.

Reply:
Sure, codecov is fine, probably too many files deleted :upside_down_face: